.gitignore

```
@@ -1,3 +1,8 @@
 .vscode
 *.tar.gz
-__pycache__
\ No newline at end of file
+__pycache__
+# Ignore specific Ansible files
+playbook.yml
+inventory.ini
+ansible.cfg
+8.1.24.0-TIV-TSMBAC-LinuxX86.tar
```

Probably best to just be generic here e.g.
```suggestion
*.tar
* .tar.gz
```

---

Hey @Tompage1994 i have done the necessary changes. Please review and confirm if this resolves the issue